src/test/ui/keyword_generics/feature-gate-keyword_generics.rs

```diff
@@ -0,0 +1,6 @@
+// This is a fake feature gate test. There is currently no way
+// to fail a test only with the feature gate missing.
+
+fn main() {
+    "just fail this test" //~ ERROR mismatched types
+}
```

Comment:
Add a `FIXME` note to change this to an actual feature gate test if this fixes bugs or introduces new syntax.